### DISCUSSION TITLE
Different SDK's for Android and IOS

### DISCUSSION THREAD

**Discussion by cnbrown04** [skiptools/skip#234] [Q&A] [answered]
I'm currently developing an iOS app that uses a SDK that has two separate versions for iOS and Android. Does Skip translate that to Kotlin or can I use the android sdk as well. 

Specifically, I am using the Zebra RFID SDK which integrates their RFID scanners using bluetooth. I wanted to see about using Skip instead of something like React Native.

**Comment by aabewhite** [comment, marked as answer]
In this situation you should use the approach outlined here to use the separate versions: https://skip.tools/docs/dependencies/#implementation
